src/Api/Settings.php

```diff
@@ -0,0 +1,48 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Billogram\Api;
+
+use Billogram\Model\Setting\Setting;
+
+class Settings extends HttpApi
```

Comment:
Why plural here and not in `BillogramClient::setting()`? Should it be plural or singular?